microprofile/microprofile-config-1.3/src/main/java/org/wildfly/swarm/ts/microprofile/config/v13/HelloServlet.java

```diff
@@ -0,0 +1,50 @@
+package org.wildfly.swarm.ts.microprofile.config.v13;
+
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.inject.Inject;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.net.URI;
+import java.util.Optional;
+
+@WebServlet("/")
+public class HelloServlet extends HttpServlet {
+
+    @Inject
+    private Config config;
+
+    @Inject
+    @ConfigProperty(name = "my.notexisting.string.property")
+    private Optional<String> notExisting;
+
+    @Inject
+    @ConfigProperty(name = "my.existing.string.property")
+    private Optional<String> existing;
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+        // Check mapping rule: from config property name to environment variable
+        // https://github.com/eclipse/microprofile-config/issues/264
+        final String key = "SSH_AGENT_PID";
```

Comment:
I wouldn't count on any particular env var being present. Maven Surefire plugin should be able to set one for the duration of the test: https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html